packages/odyssey-react-mui/src/Dialog.tsx

```diff
@@ -134,14 +150,8 @@ const Dialog = ({
         {content}
       </DialogContent>
 
-      {(primaryCallToActionComponent ||
-        secondaryCallToActionComponent ||
-        tertiaryCallToActionComponent) && (
-        <DialogActions>
-          {tertiaryCallToActionComponent}
-          {secondaryCallToActionComponent}
-          {primaryCallToActionComponent}
-        </DialogActions>
+      {actionButtons.length > 0 && (
+        <DialogActions>{actionButtons}</DialogActions>
```

Comment:
Smart! Doesn't work if you have both sets though. Do we want to pick one set of them?